frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficSplittingModal.scss

```diff
@@ -0,0 +1,12 @@
+.odc-traffic-splitting-modal {
+  .odc-multi-column-field__col {
+    flex: none;
+    width: 10%;
+  }
+  .odc-multi-column-field__col:nth-of-type(2) {
+    width: 20%;
+  }
+  .odc-multi-column-field__col:nth-of-type(3) {
+    width: 60%;
+  }
+}
```

Comment:
This is not an appropriate solution @vikram-raj - this is a hack at best. For one, your percentages don't equal 100. 

I think we need to update the `MultiColumnField` component to support this logic between "splitting the space equally with everyone". Otherwise we'll end up with this problem on every use of this component moving forward as well.